examples/jsm/postprocessing/RenderPass.d.ts

```diff
@@ -11,8 +11,8 @@ export class RenderPass extends Pass {
   constructor(scene: Scene, camera: Camera, overrideMaterial: Material, clearColor: Color, clearAlpha: number);
   scene: Scene;
   camera: Camera;
-  overrideMaterial: Material;
-  clearColor: Color;
-  clearAlpha: number;
-  clearDepth: boolean;
+  overrideMaterial?: Material;
```

Comment:
. These are properties of the class and no optional parameters. You have to make this changes to the ctor.